modules/veteran_verification/app/controllers/veteran_verification/v0/service_history_controller.rb

```diff
@@ -1,17 +1,37 @@
 # frozen_string_literal: true
 
 require_dependency 'veteran_verification/application_controller'
+require_dependency 'notary'
 
 module VeteranVerification
   module V0
     class ServiceHistoryController < ApplicationController
+      include ActionController::MimeResponds
+
+      NOTARY = VeteranVerification::Notary.new(Settings.vet_verification.key_path)
+
       before_action { authorize :emis, :access? }
       before_action { permit_scopes %w[service_history.read] }
+      before_action { set_default_format }
 
       def index
         response = ServiceHistoryEpisode.for_user(@current_user)
 
-        render json: response, each_serializer: VeteranVerification::ServiceHistorySerializer
+        serialized = ActiveModelSerializers::SerializableResource.new(
+          response,
+          each_serializer: VeteranVerification::ServiceHistorySerializer
+        )
+
+        respond_to do |format|
+          format.json { render json: serialized.to_json }
+          format.jwt { render body: NOTARY.sign(serialized.serializable_hash) }
+        end
+      end
+
+      private
+
+      def set_default_format
```

Comment:
Could you add your explanation for using this method (rather than the route default) as a comment?

Reply:
Also, I think this method could be extracted to the `ApplicationController` if you want!

Reply:
Moved this to the ApplicationController as suggested